File: NetworkToolsBckup/src/main/java/com/wifinet/internetcheck/dialogs/BottomDialogCustom.java
/*
package com.wifinet.internetcheck.dialogs;

import android.app.Dialog;
import android.os.Build;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.EditText;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.annotation.RequiresApi;

import com.google.android.material.bottomsheet.BottomSheetDialogFragment;
import com.google.android.material.button.MaterialButton;
import com.thanosfisherman.wifiutils.WifiUtils;
import com.wifinet.internetcheck.R;

public class BottomDialogCustom extends BottomSheetDialogFragment {

    private String mac;

    public BottomDialogCustom (String mac) {

        this.mac = mac;
    }


    public BottomDialogCustom () {

    }


    DialogFullScreen dialogFullScreen;

    @NonNull
    @Override
    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
        return super.onCreateDialog(savedInstanceState);
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.bottom_dialog_custom_wps,container,false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {

        EditText editText = view.findViewById(R.id.edit);
        MaterialButton btn_connect = view.findViewById(R.id.btn_connect);
        MaterialButton btn_cancel = view.findViewById(R.id.btn_cancel);
        dialogFullScreen = new DialogFullScreen();

        btn_cancel.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                dismiss();
            }
        });

        btn_connect.setOnClickListener(new View.OnClickListener() {
            @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
            @Override
            public void onClick(View v) {

                dialogFullScreen.show(getParentFragmentManager(),dialogFullScreen.getTag());

                WifiUtils.withContext(requireContext())
                        .connectWithWps(mac, editText.getText().toString())
                        .onConnectionWpsResult(BottomDialogCustom.this::checkResult)
                        .start();

            }
        });

    }

    private void checkResult(boolean isSuccess)
    {
        dialogFullScreen.dismiss();
        if (isSuccess)
            Toast.makeText(requireContext(), "Succeed", Toast.LENGTH_SHORT).show();
        else
            Toast.makeText(requireContext(), "Failed", Toast.LENGTH_SHORT).show();
    }

}*/

File: Cleaner/src/main/java/com/kerols/phoneboost/Fragments/BottomDialogCustom.kt
package com.kerols.phoneboost.Fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.RadioButton
import android.widget.RadioGroup
import com.google.android.material.bottomsheet.BottomSheetDialogFragment
import com.kerols.phoneboost.R

class BottomDialogCustom(var mOnClickRadio : com.kerols.phoneboost.Fragments.BottomDialogCustom.OnClickRadio, var str : String)  : BottomSheetDialogFragment() {


    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.bottom, container, false)
    }



    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        val mRadios = view.findViewById<RadioGroup>(R.id.radioGroup)
        val big = view.findViewById<RadioButton>(R.id.radioBig)
        val small = view.findViewById<RadioButton>(R.id.radioSmall)
        val last = view.findViewById<RadioButton>(R.id.radioLast)

        if (str == "small"){
              small.isChecked = true
        }
        else if (str == "big") {
              big.isChecked = true
        }
        else {
             last.isChecked = true
        }

        mRadios.setOnCheckedChangeListener { group, checkedId ->
               if (checkedId == R.id.radioSmall) {
                     mOnClickRadio.onEventClick("small",this)
            } else if (checkedId == R.id.radioLast) {
                     mOnClickRadio.onEventClick("last",this)
            } else {
                     mOnClickRadio.onEventClick("big",this)
                }

        }
    }

    interface OnClickRadio {
        fun onEventClick(string: String , it  : com.kerols.phoneboost.Fragments.BottomDialogCustom)
    }

}